ci/integration/metadata-ca-cert-request-integration.yaml

```diff
@@ -8,7 +8,7 @@ metadata:
 spec:
   countryCode: GB
   commonName: Proxy Node Integration Metadata CA
-  expiryMonths: 120
```

Comment:
The PR comment mentions that prod CA should have an expiry of 5 years. This updates integration but looks like it leaves prod at 10?

Reply:
Let me double check that!